app/mailers/audit_mailer.rb

```diff
@@ -0,0 +1,19 @@
+class AuditMailer < ApplicationMailer
+  helper :audits, :layout
+
+  def summary(options = {})
+    raise ::Foreman::Exception.new(N_("Must specify a user with email enabled")) unless (user=User.find(options[:user])) && user.mail_enabled?
+    time = options[:time] ? %(time > "#{options[:time]}") : 'time > yesterday'
```

Comment:
Note, leave `time > yesterday` for the reason noted in the second paragraph, it works properly with day comparisons, just change the first timestamp based search.